.github/workflows/msal-node-e2e.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: msal-node E2E Tests
+
+on:
+  push:
+    branches: 
+      - dev
+      - master
+  pull_request:
+    paths: 
+      - 'msal-node/**/*' 
+      - 'msal-common/**/*'
+      - 'samples/msal-node-samples/EndToEndTestApp/**/*'
+
+jobs:
+  run-e2e:
+    if: (github.repository == 'AzureAD/microsoft-authentication-library-for-js') && ((github.event.pull_request.head.repo.full_name == github.repository) || (github.event_name == 'push'))
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+
+    - name: Clean Install
+      env:
+        RUNNING_NODE_CI: 1
+      run: npm ci
+
+    - name: Install Dependencies
+      working-directory: samples/msal-node-samples/EndToEndTestApp
+      run: npm install
```

Comment:
Does this still work if you use `npm` ci here?

Reply:
It does, should I try removing the root-level npm ci step and see if that still works?

Reply:
You'll still need the root level one to build the packages, this should be fine